prov/efa/src/rxr/rxr_init.c

```diff
@@ -75,7 +75,7 @@ struct rxr_env rxr_env = {
 	.efa_min_read_msg_size = 1048576,
 	.efa_min_read_write_size = 65536,
 	.efa_read_segment_size = 1073741824,
-	.rnr_retry = 3,
+	.rnr_retry = EFA_RNR_INFINITE_RETRY,
```

Comment:
Like I said in our meeting, add a comment in the code about why this is set to INFINITE_RETRY, so that someone can figure it out later.

Reply:
Sorry, I thought you meant the commit message. I will add in the code.